src/instructlab/configuration.py

```diff
@@ -97,6 +98,10 @@ def _reset(self):
     def CHECKPOINTS_DIR(self) -> str:
         return path.join(self._cache_home, STORAGE_DIR_NAMES.CHECKPOINTS)
 
+    @property
```

Comment:
is this added to the list of dirs necessary upon `init`? I think there is a list somewhere

Reply:
sounds like you're referring to https://github.com/instructlab/instructlab/blob/main/src/instructlab/configuration.py#L525

I can add it there, though I'm creating that directory anyway so it wouldn't matter 
I don't know if we want to stick to creating all directories at initialization or just create them as and when we need them in the code